### PR TITLE
[Catalog] Wire up the entity presentation API in new frontend system apps

### PR DESCRIPTION
**Pull request by iamEAP**
## Hey, I just made a Pull Request!

Probably just an oversight?

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/catalog/src/alpha/apis.tsx [hunk 1]
```diff
@@ -48,4 +52,13 @@ export const catalogStarredEntitiesApi = createApiExtension({
   }),
 });
 
-export default [catalogApi, catalogStarredEntitiesApi];
+export const entityPresentationApi = createApiExtension({
+  factory: createApiFactory({
+    api: entityPresentationApiRef,
+    deps: { catalogApiImp: catalogApiRef },
+    factory: ({ catalogApiImp }) =>
+      DefaultEntityPresentationApi.create({ catalogApi: catalogApiImp }),
```
Comment (freben) [inline plugins/catalog/src/alpha/apis.tsx:60]:
```suggestion
    deps: { catalogApi: catalogApiRef },
    factory: ({ catalogApi }) =>
      DefaultEntityPresentationApi.create({ catalogApi }),
```

just for uniformity

**Review by freben — APPROVED** [review]
Nice, thanks. @backstage/catalog-maintainers if you are good for the release tomorrow?

File: plugins/catalog/src/alpha/apis.tsx (hunk 1 above)
Comment (iamEAP) [reply plugins/catalog/src/alpha/apis.tsx:60]: Yeaaaah, but `catalogApi` is defined in the parent scope.

Comment (Rugvip) [reply plugins/catalog/src/alpha/apis.tsx:60]: Well that's a bug in our naming convention 😅, making a note of it and this is obv fine to ship as is 👍

**Review by Rugvip — APPROVED** [review]

**Review by vinzscam — APPROVED** [review]

**Pull request merged**